src/components/dialog/dialog.test.js

```diff
@@ -274,6 +285,10 @@ context("Testing Dialog component", () => {
       cy.get("body").tab();
       getInput(2).should("be.focused");
       cy.get("body").tab();
+      textEditorInput().should("be.focused");
```

Comment:
similarly to Kuba's comment on the stories, I'd rather add a dedicated test for this than just add it to an existing big test - to make it clear we've got a specific regression test for TextEditor inside a Dialog. I'd also make a new component in this file for testing this rather than hijacking DialogComponent, but I'm happy to leave that up to you, you may disagree.